### PR TITLE
Add Erlanger Stadtwerke (ESTW)

### PR DESCRIPTION
**Pull request by nodeg**

### CODE REVIEW

File: companies/estw.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "slug": "estw",
+    "relevant-countries": [
+        "de"
+    ],
+    "name": "ESTW - Erlanger Stadtwerke AG",
+    "address": "Äußere Brucker Straße 33\n91052 Erlangen",
```
Comment (baltpeter) [inline companies/estw.json]: `Deutschland`

File: companies/estw.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "phone": "+49 9131 8230",
+    "fax": "+49 9131 8234733",
+    "email": "datenschutz@estw.de",
+    "web": "https://www.estw.de",
+    "sources": [
+        "https://www.estw.de/de/Home/Impressum",
+        "https://www.estw.de/de/Home/Datenschutz/Rechte-Seite/informationen-datenschutz-2019-v1-01.pdf"
+    ]
+}
```
Comment (baltpeter) [inline companies/estw.json]: Please also add a suggested transport medium here.

**Comment by nodeg** [comment]
Thanks for your comments and sorry for the little mess.

**Comment by baltpeter** [comment]
Thanks, looks good now.

Not sure if you've seen this already but feel free to signup to our Hacktoberfest event to get some stickers from us, if you want: https://www.datarequests.org/blog/hacktoberfest-2019/

**Pull request merged**